test/helpers/base-steps/simulate-server.js

```diff
@@ -1,26 +1,26 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import { test } from '@bigtest/suite';
 import { bigtestGlobals } from '@bigtest/globals';
 import localforage from 'localforage';
 import { start } from '../server';
 
 bigtestGlobals.defaultInteractorTimeout = 12_000;
 
-export default function simulateServer(name, { permissions = [] } = { permissions: [] }) {
+export default function simulateServer(name, { permissions = [] } = { permissions: [] }, user) {
   return test(name)
     .step('set up localforage', async () => {
       localforage.clear();
       localforage.setItem('okapiSess', {
```

Comment:
localforage is an async API and so in order to be correct, we need an `await` here. Otherwise, there is a potential race condition.
```suggestion
      await localforage.setItem('okapiSess', {
```